src/components/top_nav/TopNav.module.scss

```diff
@@ -1,131 +1,211 @@
 @import '../../utils/variables';
 
-.navContainer {
-  width: 100%;
-  height: auto;
-  z-index: 20;
-  background: transparent;
-  box-sizing: border-box;
-  max-height: 100px;
-  top: 0;
-  position: sticky;
-  padding: 12px 18px;
-  box-shadow: $boxShadow;
-  background: $white;
-}
-.navContent {
-  display: flex;
-  justify-content: space-between;
-  margin: auto;
-  position: relative;
-  align-items: center;
-}
-.logoContainer {
-  display: flex;
-  width: 25vh;
-  height: fit-content;
-  max-width: 240px;
-  min-width: 150px;
-  border-radius: 10px;
-}
-.logo {
-  width: 100%;
-}
-.navigation {
-  display: flex;
-  max-width: 800px;
-  width: calc(100% - 250px);
-}
-.navRow {
-  align-items: center;
-  width: 100%;
-  justify-content: space-between;
-  display: flex;
-  list-style: none;
-  box-sizing: border-box;
-}
-.navLink {
-  font-weight: $semi-bold;
-  color: $blue;
-  font-size: calc(14px + (24 - 14) * ((100vw - 320px) / (1920 - 320)));
-  transition: color 0.4s;
-  &:hover {
-    color: $brand;
-  }
-}
-.navIcon {
-  display: none;
-}
+.fullNavContainer {
+    position: sticky;
+    z-index: 20;
+    top: 0;
+    box-shadow: $boxShadow;
 
-.mobileMenu {
-  display: flex;
-  flex-direction: column;
-  position: absolute;
-  background: $white;
-  width: 100%;
-  left: 0;
-  color: $blue;
-  height: fit-content;
-  box-sizing: border-box;
-  box-shadow: $boxShadow;
-  border-radius: 10px;
-  visibility: hidden;
-  transform: translateY(-150%);
-  top: 0;
-  transition: all 0.4s ease;
-  padding: 1em;
-  & i {
-    font-size: 24px;
-    margin-left: auto;
-  }
-}
-.mobileMenuHeader {
-  display: flex;
-  justify-content: space-between;
-  margin-bottom: 1em;
-  & .logo {
-    width: 150px;
-  }
-}
-.mobileNavLink {
-  line-height: 2.5;
-  color: $blue;
-  text-decoration: none;
-}
+    //this is the navbar on top
+    .topNavContainer {
+        width: 100%;
+        height: auto;
+        box-sizing: border-box;
+        max-height: 100px;
+        padding: 12px 18px;
+        background: $topNavColor;
+
+        @media (max-width: $medium) {
+            display: none;
+        }
 
-@media (max-width: $medium) {
-  .navIcon {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    height: fit-content;
-    cursor: pointer;
-    width: 24px;
-    margin-left: auto;
-    margin-top: 0.5em;
-    & > div {
-      transform-origin: left;
-      height: 3px;
-      width: inherit;
-      background: $blue;
-      transition: transform 0.7s;
-      margin-bottom: 6px;
+        .topNavRow {
+            align-items: center;
+            width: 100%;
+            justify-content: flex-end;
+            display: flex;
+            list-style: none;
+            box-sizing: border-box;
+
+            @media (max-width: $medium) {
+                display: none;
+            }
```

Comment:
If you look at the topNavContainer, the display has already been set to none for this screen size so I dont think you need to put it here, too?